### PR TITLE
Set up translation infrastructure

### LINKED ISSUE #2: Set up translation infrastructure
Right now there is no infrastructure to extract the strings and perform translations. Please add this.

### PR DESCRIPTION
**Pull request by stejskalleos**
Fixing https://github.com/theforeman/hammer-cli-foreman-google/issues/2

### CODE REVIEW

**Review by ofedoren — APPROVED** [review]
Thanks, @stejskalleos, seems to be right.

File: lib/hammer_cli_foreman_google/i18n.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require 'hammer_cli/i18n'
+
+module HammerCLIForemanGoogle
+  module I18n
+
+    class LocaleDomain < HammerCLI::I18n::LocaleDomain
+
+      def translated_files
+        Dir.glob(File.join(File.dirname(__FILE__), '../**/*.rb'))
```
Comment (ekohl) [inline lib/hammer_cli_foreman_google/i18n.rb]:
```suggestion
        Dir.glob(File.join(__dir__, '../**/*.rb'))
```

File: lib/hammer_cli_foreman_google/i18n.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      end
+
+      def locale_dir
+        File.join(File.dirname(__FILE__), '../../locale')
```
Comment (ekohl) [inline lib/hammer_cli_foreman_google/i18n.rb]:
```suggestion
        File.join(__dir__, '../../locale')
```

**Review by ekohl — APPROVED** [review]
Looks correct. Some very minor things that Rubocop will also find and suggest to you.

**Pull request merged**